src/main/java/frc/robot/math/Vec2.java

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.math;
+
+import edu.wpi.first.math.Matrix;
+import edu.wpi.first.math.Vector;
+import edu.wpi.first.math.numbers.N1;
+import edu.wpi.first.math.numbers.N2;
+import java.util.function.DoubleUnaryOperator;
+import org.ejml.simple.SimpleMatrix;
+
+/** Vector in 2 dimensions */
+public class Vec2 extends Vector<N2> {
```

Comment:
What's this class for? Just a convenience wrapper for Vector<N2>?

Reply:
Replaces MoveVector (which represented fwd and left request, and handled applying curves), and because it implements the ejml vector wrapper we can do any other kind of math more easily with it